integration-test/bin/cdk.ts

```diff
@@ -3,4 +3,5 @@ import { App } from "@aws-cdk/core";
 import { IntegrationTestStack } from "../src/integration-test-stack";
 
 const app = new App();
-new IntegrationTestStack(app, "Cdk", { stack: "integration-test" });
+new IntegrationTestStack(app, "IntegrationTestStackCODE", { stack: "integration-test-CODE" });
+new IntegrationTestStack(app, "IntegrationTestStackPROD", { stack: "integration-test-PROD" });
```

Comment:
We synthesise these stacks [in CI](https://github.com/guardian/cdk/blob/aeebaa8f60f5f3df08958686106be088903954a4/integration-test/package.json#L16), this step will exit non-zero on failure.